README.md

```diff
@@ -0,0 +1,40 @@
+Jinja2 extension to handle git-specific things
+==============================================
```

Comment:
This should be `# Jinja2 extension to handle git-specific things`, without `===` part